multi/multi.go

```diff
@@ -115,9 +136,19 @@ func (connMulti *ConnectionMulti) deleteConnectionFromPool(addr string) {
 	delete(connMulti.pool, addr)
 }
 
+func (connMulti *ConnectionMulti) clearConnectionPool() {
+	connMulti.mutex.Lock()
+	defer connMulti.mutex.Unlock()
+	for k := range connMulti.pool {
+		delete(connMulti.pool, k)
+	}
+}
+
 func (connMulti *ConnectionMulti) checker() {
+
 	for connMulti.getState() != connClosed {
-		timer := time.NewTimer(connMulti.opts.CheckTimeout)
+		checkTimer := time.NewTimer(connMulti.opts.CheckTimeout)
```

Comment:
was variable name important for this PR?

Reply:
Sorry, missed it. Cleaned from commit.